examples/day_in_history_template.js

```diff
@@ -11,9 +11,9 @@ const structure_template = (`
 <section class="mainContent">
 
   <h3>On this Day in History</h3>
-    <p id="dayInHistoryHeadFinal">{{ dayInHistoryHeadFinal }}</p>
-    <p id="dayInHistoryBodyFinal">{{ dayInHistoryBodyFinal }}</p>
-    <p id="dayInHistoryYearFinal">{{ dayInHistoryYearFinal }}</p>
+    <p id="dayInHistoryHead">{{ dayInHistoryHead }}</p>
+    <p id="dayInHistoryBody">{{ dayInHistoryBody }}</p>
+    <p id="dayInHistoryYear">{{ dayInHistoryYear }}</p>
```

Comment:
Generally, messing with other non-related functionality would be in separate commits but not a blocker for now.